examples/stacks/lepp-stack/devbox.json

```diff
@@ -22,6 +23,7 @@
       "run_test": [
         "mkdir -p /tmp/devbox/lepp",
         "export PGHOST=/tmp/devbox/lepp",
+        "rm -rf .devbox/virtenv/postgresql/data",
```

Comment:
why is this needed?

Reply:
This just makes testing easier by making `run_test` more idempotent.